Paymetheus.Rpc/WalletClient.cs

```diff
@@ -314,6 +332,77 @@ public void CancelRequests()
             return Tuple.Create(signedTransaction, complete);
         }
 
+        public async Task<List<Blake256Hash>> PurchaseTicketsAsync(Account account, Amount spendLimit, 
+            int reqConfs, Address ticketAddress, uint number, Address poolAddress, double poolFees, 
+            uint expiry, Amount txFee, Amount ticketFee, string passphrase)
```

Comment:
This function is gross. can we either split it into multiple functions depending on exactly how it's being used (with/without a pool, with/without the ticket address)?